app/assets/javascripts/discourse/app/components/edit-category-topic-template.js

```diff
@@ -1,15 +1,51 @@
 import { buildCategoryPanel } from "discourse/components/edit-category-panel";
-import { observes } from "discourse-common/utils/decorators";
+import discourseComputed, { observes } from "discourse-common/utils/decorators";
 import { schedule } from "@ember/runloop";
+import FormTemplate from "admin/models/form-template";
+import { action } from "@ember/object";
+import { notEmpty } from "@ember/object/computed";
 
 export default buildCategoryPanel("topic-template", {
   // Modals are defined using the singleton pattern.
   // Opening the insert link modal will destroy the edit category modal.
   showInsertLinkButton: false,
+  showFormTemplate: notEmpty("category.form_template_ids"),
 
-  @observes("activeTab")
+  init() {
+    this._super(...arguments);
+
+    FormTemplate.findAll().then((result) => {
+      const sortedTemplates = this._sortTemplatesByName(result);
+      this.set("templates", sortedTemplates);
+    });
```

Comment:
I understand it's not the best approach to do the initial fetching and setting of form templates inside `init()` method, but I tried the alternate of creating a select kit addon so the request is triggered when the dropdown is clicked but ran into some issues with promises and the label not setting correctly. Since we're in the admin panel, performance isn't as crucial and I don't want this to be a blocker for continuing work on this feature as we can revisit this again later, so I've left it as it is.